aws-env.dev

```diff
@@ -1,13 +1,13 @@
-export BUGZILLA_ADMIN_KEY=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
-export BUGZILLA_URL=https://dashboard.dev.unee-t.com
-export MONGO_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export MONGO_CONNECT="dev-shard-00-00-qxxao.mongodb.net:27017,dev-shard-00-01-qxxao.mongodb.net:27017,dev-shard-00-02-qxxao.mongodb.net:27017/test?ssl=true&replicaSet=Dev-shard-0&authSource=admin"
-export MYSQL_DATABASE=bugzilla
-export MYSQL_HOST=auroradb.dev.unee-t.com
-export MYSQL_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export MYSQL_PORT=3306
-export MYSQL_USER=bugzilla
-export PARAMS_URL=https://raw.githubusercontent.com/unee-t/bugzilla-customisation/master/params/dev-params.json
-export SES_SMTP_PASSWORD=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_PASSWORD --with-decryption --query Parameters[0].Value --output text)
-export SES_SMTP_USERNAME=$(aws --profile uneet-dev ssm get-parameters --names SES_SMTP_USERNAME --query Parameters[0].Value --output text)
-export SES_VERIFIED_SENDER=dev.case@unee-t.com
+export BUGZILLA_ADMIN_KEY_INS=$(aws --profile uneet-dev ssm get-parameters --names BUGZILLA_ADMIN_KEY --with-decryption --query Parameters[0].Value --output text)
+export BUGZILLA_URL_INS=https://dashboard-ins.dev.unee-t.com
+export MONGO_PASSWORD_INS=$(aws --profile uneet-dev ssm get-parameters --names MONGO_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export MONGO_CONNECT_INS="dev-shard-00-00-qxxao.mongodb.net:27017,dev-shard-00-01-qxxao.mongodb.net:27017,dev-shard-00-02-qxxao.mongodb.net:27017/test?ssl=true&replicaSet=Dev-shard-0&authSource=admin"
+export MYSQL_DATABASE_INS=bugzilla
+export MYSQL_HOST_INS=auroradb-ins.dev.unee-t.com
+export MYSQL_PASSWORD_INS=$(aws --profile uneet-dev ssm get-parameters --names MYSQL_PASSWORD --with-decryption --query Parameters[0].Value --output text)
+export MYSQL_PORT_INS=3306
+export MYSQL_USER_INS=bugzilla
+export PARAMS_URL_INS=https://github.com/tuananh1508/bugzilla-customisation/blob/master/params/dev-params.json
```

Comment:
assume that we will run the code from the master <-- use the url for the version of the file in the master repo, not the version of the URL from your fork.